src/client/components/ui/Carousel/Slide.tsx

```diff
@@ -48,6 +48,7 @@ export const Slide = (props: SlideProps) => {
 			aria-current={isCurrentSlide}
 			aria-hidden={!isCurrentSlide}
 			aria-roledescription="slide"
+			aria-label={`Image ${index + 1} sur ${numberOfImages}`}
```

Comment:
J'ai un gros doute sur l'interprétation de cette partie : 

> Each slide container has role [group](https://w3c.github.io/aria/#group) with the property [aria-roledescription](https://w3c.github.io/aria/#aria-roledescription) set to slide.
Each slide has an accessible name:
If a slide has a visible label, its accessible label is provided by the property [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) on the slide container set to the ID of the element containing the visible label.
Otherwise, an accessible label is provided by the property [aria-label](https://w3c.github.io/aria/#aria-label) set on the slide container.
If unique names that identify the slide content are not available, a number and set size can serve as a meaningful alternative, e.g., "3 of 10". Note: Normally, including set position and size information in an accessible name is not appropriate. An exception is helpful in this implementation because group elements do not support [aria-setsize](https://w3c.github.io/aria/#aria-setsize) or [aria-posinset](https://w3c.github.io/aria/#aria-posinset). The tabbed carousel implementation pattern does not have this limitation.
Note that since the aria-roledescription is set to "slide", the label does not contain the word "slide."

du coup est ce qu'on a besoin de ce aria-label ? est ce qu'il manque pas le rôle de aria-roledescription set à `slide`?